### ISSUE TITLE
Should countables be coercible to sets?

### ISSUE THREAD

**Issue by helins** [Convex-Dev/convex#345]
`vec` can take countables (strings, blobs, etc) while `set` takes only  collections. Is it by design or rather an omission?

**Comment by mikera** [comment]
It a bit questionable whether you should be building sets from countables. In particular it might be expensive on juice, since you would need to hash each element. But in theory we could support it.

**Comment by helins** [comment]
Exactly, for the sake of being complete and consistent (signs of a good language).
